cognite/neat/_rules/models/information/_rules.py

```diff
@@ -103,9 +101,6 @@ def remove_default_prefixes(self, value: Any, info: SerializationInfo) -> str:
             )
         return ",".join(str(value) for value in value)
 
-    def as_reference(self) -> ClassRef:
```

Comment:
Is there a reason why we are removing this? I guess we are not using it or ?

Reply:
It is connected to the Mapping class which I removed since we are now using DMS rules for mapping instaed.